src/AuthenticationPortal.Contracts/Errors/BaseException.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+
+namespace AuthenticationPortal.Contracts
+{
+    [Serializable]
+    public class BaseException : Exception
+    {
+        public int Code { get; set; }
+        public string Message { get; set; }
+        public List<ErrorInfo> Info { get; set; }
+        public HttpStatusCode StatusCode { get; set; }
+        public BaseException(string message, int code, List<ErrorInfo> info, HttpStatusCode statusCode) : base(message)
```

Comment:
why there are setters for all properties if you have a ctor for accepting all the prop?
also constructor should only take the arguments which are mandatory

Reply:
Done